modules/vpc/main.tf

```diff
@@ -2,6 +2,13 @@ module "vpc" {
   source  = "terraform-aws-modules/vpc/aws"
   version = "5.1.2"
 
-  name = local.vpc_name
-  cidr = var.cidr
+  name                   = local.vpc_name
+  cidr                   = var.cidr
+  azs                    = local.azs
```

Comment:
not sure about that, based on the diagram, I explicitly defined 2 AZs we are using, so I think it's fair to put them in the `locals` and use them directly.
Do you think the template left the `available` in empty because we need to put the specific AZ names when using it?

Reply:
It's `data source`. If the block is empty it will get all available AZs from AWS in that region you have selected. Which is probably `ap-southeast-1a`, `ap-southeast-1b`, `ap-southeast-1c`

>  I explicitly defined 2 AZs we are using, so I think it's fair to put them in the locals and use them directly.

Sure, I will leave this comment here in case Terone has anything to add when he reviews it.